<launch>
	<arg 
		name="device_name"
		default="kern_balance"/>
	<arg
		name="connection_mode"
		default="serial"/>
	<arg
		name="adress"
		default="None"/>
	<arg
		name="port"
		default="/dev/ttyUSB0"/>
	<node 
		name="kern_pcb_top_balance"
		pkg="kern_pcb_top_balance"
		type="kern_pcb_top_balance"
		args="
			$(arg device_name)
			$(arg connection_mode)
			$(arg address)
			$(arg port)"
		output="screen"/>
</launch>
